### PR TITLE
10012_nr_affliation_emailcase

### PR DESCRIPTION
**Pull request by ozamani9**
*Issue #:*
https://github.com/bcgov/entity/issues/10012

*Description of changes:*

add casefold function to both the applicants email and the verification email in business affiliation logic.

### CODE REVIEW

File: auth-api/src/auth_api/version.py [hunk 1]
```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '0.1.0a0.dev'  # pylint: disable=invalid-name
+__version__ = '0.2.0a0'  # pylint: disable=invalid-name
```
Comment (saravanpa-aot) [inline auth-api/src/auth_api/version.py:25]: I am not sure if we need to do this.. any particular reason for doing this?

File: auth-api/tests/unit/services/test_affiliation.py [hunk 2]
```diff
@@ -344,7 +344,7 @@ def test_create_new_business(session, auth_mock, nr_mock):  # pylint:disable=unu
     business_identifier = 'NR 1234567'
 
     affiliation = AffiliationService.create_new_business_affiliation(org_id, business_identifier=business_identifier,
-                                                                     email='test@test.com', phone='1112223333')
+                                                                     email='TEST@test.com', phone='1112223333')
```
Comment (saravanpa-aot) [inline auth-api/tests/unit/services/test_affiliation.py:347]: if you want , you could a different test case with both small case and capital.. that will be more clear and helpful

Comment (ozamani9) [reply auth-api/tests/unit/services/test_affiliation.py:347]: ok i will create two tests.

File: auth-api/src/auth_api/version.py (hunk 1 above)
Comment (saravanpa-aot) [reply auth-api/src/auth_api/version.py:25]: Yeah...I am aware of that..
We release a bit different from entities ;mostly big release we do with bunch of tickets together..We don't generally rollout one change into prod quickly..mostly big releases...So we don't have this practice so far..but i am okay to bring in this practice ...May be we can discuss with @pwei1018 and agree on it..

Comment (ozamani9) [reply auth-api/src/auth_api/version.py]: fixed version and a lint error and repushing now

**Review by pwei1018 — APPROVED** [review]

**Pull request merged**